tests/cupy_tests/math_tests/test_misc.py

```diff
@@ -132,6 +132,19 @@ def test_external_clip2(self, xp, dtype):
         a = testing.shaped_arange((2, 3, 4), xp, dtype)
         return xp.clip(a, 3, 13)
 
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_array_equal()
+    def test_external_clip3(self, xp, dtype):
+        a = testing.shaped_arange((2, 3, 4), xp, dtype)
+        return xp.clip(a, 8, 1)
```

Comment:
Could you fix the test to check also for the case of `x[i] < a_max`?
```suggestion
        return xp.clip(a, 8, 4)
```